src/drivers/pwm_out/PWMOut.cpp

```diff
@@ -76,14 +78,16 @@ int PWMOut::init()
 		PX4_ERR("FAILED registering class device");
 	}
 
+	// Legacy mixer-module operation flag
+	_legacy_mixer_mode = (_p_pwm_aux_mode.get() == 0) ? true : false;
```

Comment:
I'll probably delete this - I was thinking there could be some utility in a sort of mode-switch, but I don't think it has a use case for the current implementation.